### ISSUE TITLE
🔹[EPIC] Migrate Pagoda docs to docs.near.org or deprecate

### ISSUE THREAD

**Issue by thisisjoshford** [near/docs#1547]
### Description
docs.pagoda.co should have all relevant documentation migrated to docs.near.org for better visibility. Anything that is no longer relevant can be deprecated.

Current docs on docs.pagoda.co:
- Pagoda RPC
- Console alerts & triggers
- Pagoda NEAR Lake
- Enhances API

[Slack Message](https://pagodaplatform.slack.com/archives/C03PKHB82QP/p1697579432239139?thread_ts=1697566746.085089&cid=C03PKHB82QP)
```[tasklist]
### Tasks
- [ ] https://github.com/near/docs/issues/1775
- [ ] https://github.com/near/docs/issues/1776
```

**Comment by bucanero** [comment]
hi @charleslavon  👋 

the PR is done and ready to merge: https://github.com/near/docs/pull/1774

The issue/concern was that by moving Pagoda's RPC docs, we are giving Pagoda an "advantage" over other external RPC providers. Perhaps you can chat with Josh and define what we should do about it?

IMHO, I think the PR could be merged anyway to unblock the rest of the tasks, and deprecate `docs.pagoda.co`. Then at a later point if Pagoda makes a decision about the Enhanced API, we can update or remove that content from `docs.near.org`
